Foundation/System/Bindings/Network.hsc

```diff
@@ -0,0 +1,45 @@
+-- |
+-- Module      :  Foundation.System.Bindings.HostName
+-- License     :  BSD-style
+-- Maintainer  :  Nicolas Di Prima <nicolas@primetype.co.uk>
+-- Stability   :  provisional
+-- Portability :  portable
+--
+module Foundation.System.Bindings.Network
+    ( -- * error
+      getHErrno
+    , herr_HostNotFound
+    , herr_NoData
+    , herr_NoRecovery
+    , herr_TryAgain
+    ) where
+
+import Foundation.Internal.Base
+import Foreign.C.Types
+
+#ifdef mingw32_HOST_OS
+# include <winsock2.h>
+#else
+# include "netinet/in.h"
+# include "netdb.h"
+#endif
+
+herr_HostNotFound
```

Comment:
I see where you come from but this is not really related to `errno`. Would rather like to avoid confusion, especially for something restricted to hostname and that would certainly not have any other usage in other modules ?